tools/board_stubs/build_board_specific_stubs/board_stub_builder.py

```diff
@@ -17,9 +17,12 @@ def get_board_pins(pin_filename):
             if line.strip()[0:2] == "//":
                 continue
 
-            search = re.search(r"MP_ROM_QSTR\(MP_QSTR_(.*?)\), MP_ROM_PTR", line)
+            # (\s*) means any amount of whitespaces (no whitespaces allowed too)
+            # related issue: https://github.com/adafruit/circuitpython/issues/9407
+
+            search = re.search(r"MP_ROM_QSTR\(MP_QSTR_(.*?)\),(\s*)MP_ROM_PTR", line)
```

Comment:
Parentheses make this be a saved match group, so let's take those away.
```suggestion
            search = re.search(r"MP_ROM_QSTR\(MP_QSTR_(.*?)\),\s*MP_ROM_PTR", line)
```